crates/bevy_render/src/camera/camera.rs

```diff
@@ -530,12 +530,17 @@ pub fn extract_cameras(
             &CameraRenderGraph,
             &GlobalTransform,
             &VisibleEntities,
+            Option<&ColorGrading>,
         )>,
     >,
     primary_window: Extract<Query<Entity, With<PrimaryWindow>>>,
 ) {
     let primary_window = primary_window.iter().next();
-    for (entity, camera, camera_render_graph, transform, visible_entities) in query.iter() {
+    for (entity, camera, camera_render_graph, transform, visible_entities, color_grading) in
+        query.iter()
+    {
+        let color_grading = *color_grading.unwrap_or(&ColorGrading::default());
```

Comment:
```suggestion
        let color_grading = *color_grading.unwrap_or_default();
```

Reply:
With this I get the error:
```
the trait bound `&ColorGrading: std::default::Default` is not satisfied
the trait `std::default::Default` is implemented for `ColorGrading`
```
I think maybe because Default is implemented for ColorGrading but not for &ColorGrading?

Reply:
Ah right. That's fine then.